Source/WebKit/UIProcess/WebProcessCache.cpp

```diff
@@ -199,7 +199,7 @@ void WebProcessCache::updateCapacity(WebProcessPool& processPool)
     } else {
 #if PLATFORM(IOS_FAMILY)
         constexpr unsigned maxProcesses = 10;
-        size_t memorySize = WTF::ramSizeDisregardingJetsamLimit() / GB;
+        size_t memorySize = (WTF::ramSizeDisregardingJetsamLimit() + 256 * MB) / GB;
```

Comment:
`+ 256 * MB` looks hacky. There must be a better way to do this.

Reply:
We likely should tweak the logic below instead of "lying" about the memory size.

Reply:
That's a good point! New patch coming up.

Thanks for reviewing!